### ISSUE TITLE
Volume changing

### ISSUE THREAD

**Issue by occune** [kmikiy/SpotMenu#62]
it will be really good if I can change volume from spotmenu
what you think about this feature?

**Comment by kmikiy** [comment]
Yes, I think its a good idea. I have thought about it too.
Which volume would you prefer to be changed? Spotify's volume or system volume? (Or controls for both?). Personally I only change the system volume.

**Comment by BatyaMedic** [comment]
I think it will be better for spotify volume

**Comment by vangy1** [comment]
I agree. I want to change spotify volume many times instead of just changing system volume and I have to open spotify window for that. Same with the "save to your library", radio thumbs up/down and shuffle buttons.

**Comment by maurojuniorr** [comment]
It would be great if there was a control for the spotify volume!

**Comment by omkarkarande** [comment]
+1

**Comment by EmlJnke** [comment]
Omg volume control would make this app the absolutely best spotify menu bar controller!
